source/manual/alerts/email-alert-api-app-healthcheck-not-ok.html.md

```diff
@@ -186,6 +186,6 @@ decreased. If it's decreasing, then it means that emails are going out and
 there's probably a lot being processed. You can also check the
 [Email Alert API Metrics dashboard][dashboard].
 
-[dashboard]: https://grafana.publishing.service.gov.uk/dashboard/file/email_alert_api.json?refresh=10s&orgId=1
+[dashboard]: https://grafana.staging.govuk.digital//dashboard/file/email_alert_api.json?refresh=10s&orgId=1
```

Comment:
You've got `//` here, which will probably still work, but only one `/` is necessary.